index.bs

```diff
@@ -139,37 +136,40 @@ extensible.
 <button id="pipButton"></button>
 
 <script>
-  let pipWindow;
-  const video = document.getElementById('video');
-  const pipButton = document.getElementById('pipButton');
+  const video = document.getElementById("video");
+  const pipButton = document.getElementById("pipButton");
 
-  pipButton.addEventListener('click', function() {
-    video.requestPictureInPicture()
-    .catch(error => {
+  pipButton.addEventListener("click", async () => {
+    try {
+      await video.requestPictureInPicture();
+    } catch (error) {
       // Video failed to enter Picture-in-Picture mode.
-    });
+    }
   });
 
-  video.addEventListener('enterpictureinpicture', function(event) {
+  video.addEventListener("enterpictureinpicture", event => {
     // Video entered Picture-in-Picture mode.
-    pipWindow = event.pictureInPictureWindow;
+    const { pictureInPictureWindow: pipWindow } = event;
     updateVideoSize(pipWindow.width, pipWindow.height);
-    pipWindow.addEventListener('resize', onPipWindowResize);
+    pipWindow.addEventListener("resize", onPipWindowResize);
   });
 
-  video.addEventListener('leavepictureinpicture', function() {
+  video.addEventListener("leavepictureinpicture", event => {
```

Comment:
`event` does not contain a `pictureInPictureWindow ` object. This is why we have a `pipWindow` object declared above.

Reply:
Ohh... um... ok... this seems like an odd API design choice here... why not fire an `EnterPictureInPictureEvent` (yes, it has "Enter" on the front, boo!) if the developer still needs to access `pictureInPictureWindow`? Otherwise, the API forces the developer to keep an unnecessary reference to a variable in the global or some outer scope?

Reply:
I've filed https://github.com/w3c/picture-in-picture/issues/188 to track this issue. Thank you!